microsoft-edge/webview2/concepts/environment-controller-core.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "How the top-level classes work: CoreWebView2, Controller, Environment, and Settings"
+description: "Overview of how the top-level WebView2 classes work: CoreWebView2, Controller, Environment, and Settings."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 06/09/2022
+---
+# How the top-level classes work: CoreWebView2, Controller, Environment, and Settings
+
+The WebView2 `Controller` and `CoreWebView2` classes work as pairs.
+
+
+#### [C#](#tab/c-sharp)
+
+* [CoreWebView2 Class](https://docs.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2)
+* [CoreWebView2Controller Class](https://docs.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2controller)
+* [CoreWebView2Environment Class](https://docs.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2environment)
+* [CoreWebView2Settings Class](https://docs.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2settings)
```

Comment:
Settings not discussed in this doc
```suggestion
```

Reply:
Removed "Settings" from title & Ref links